### PR TITLE
Update examples

### PR DESCRIPTION
**Pull request by nghialv**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: examples/cloudrun/analysis/.pipe.yaml [hunk 1]
```diff
@@ -11,7 +11,18 @@ spec:
...
       - name: ANALYSIS
+        with:
+          duration: 10m
+          metrics:
+            - provider: prometheus-dev
+              interval: 2m
+              expected:
+                max: 0.01
+              query: |
+                sum by (job) (rate(http_requests_total{status=~"5.*", job="analysis-by-metrics"}[5m]))
```
Comment (nakabonne) [inline examples/cloudrun/analysis/.pipe.yaml:22]: While I think it's not common to use Prometheus for monitoring CloudRun, for now it's fine until we support Cloud Monitoring as an analysis provider.

File: examples/cloudrun/analysis/.pipe.yaml [hunk 2]
```diff
@@ -11,7 +11,18 @@ spec:
...
+                /
+                sum by (job) (rate(http_requests_total{job="analysis-by-metrics"}[5m]))
```
Comment (nakabonne) [inline examples/cloudrun/analysis/.pipe.yaml:24]: nit
```suggestion
                sum by (job) (rate(http_requests_total{status=~"5.*", job="analysis"}[5m]))
                /
                sum by (job) (rate(http_requests_total{job="analysis"}[5m]))
```

File: examples/cloudrun/analysis/.pipe.yaml (hunk 1 above)
Comment (nakabonne) [reply examples/cloudrun/analysis/.pipe.yaml:22]: Once got supported, I'm going to update here

File: examples/kubernetes/canary-patch/deployment.yaml [hunk 3]
```diff
@@ -38,20 +28,18 @@ spec:
         image: envoyproxy/envoy-alpine:v1.18.3
         imagePullPolicy: IfNotPresent
         command:
-        - envoy
+          - /bin/sh
         args:
           - -c
-          - /etc/envoy/envoy-config.yaml
+          - envoy --config-yaml "$(cat /etc/envoy/envoy-config)"
```
Comment (khanhtc1202) [inline examples/kubernetes/canary-patch/deployment.yaml:34]: 👏

File: examples/cloudrun/analysis/.pipe.yaml (hunk 1 above)
Comment (nghialv) [reply examples/cloudrun/analysis/.pipe.yaml:22]: Right. Let me update this to use HTTP check.

**Comment by nakabonne** [comment]
Nice work!
/lgtm

File: examples/cloudrun/analysis/.pipe.yaml (hunk 1 above)
Comment (nghialv) [reply examples/cloudrun/analysis/.pipe.yaml]: Fixed it!

**Comment by khanhtc1202** [comment]
Way to go 🚀 
/approve

**Pull request merged**